packages/e2e-tests/specs/experimental/block-transforms.test.js

```diff
@@ -100,7 +100,11 @@ describe( 'Block transforms', () => {
 
 	const transformStructure = {};
 	beforeAll( async () => {
-		await enableExperimentalFeatures( [ '#gutenberg-widget-experiments', '#gutenberg-menu-block' ] );
+		await enableExperimentalFeatures( [
+			'#gutenberg-widget-experiments',
+			'#gutenberg-menu-block',
+			'#gutenberg-full-site-editing',
+		] );
```

Comment:
Can we add a `disable` equivalent to make sure we clean up after this test suite?

Reply:
#18093